### PR TITLE
implemented eigvals to frontend  #22340

### PR DESCRIPTION
**Pull request by moamen20**
# PR Description 

## Related Issue 

Close #

## Checklist 

- [x] Did you add a function?
- [x] Did you add the tests?
- [x] Did you follow the steps we provided?

### CODE REVIEW

**Comment by a0m0rajab** [comment]
Hey, can you add the tests and add the related issue to the PR body?

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_linalg/test_matrix_eigenvalues.py [hunk 1]
```diff
@@ -93,6 +93,55 @@ def test_numpy_eig(
...
+        backend_to_test=backend_fw,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        test_values=False,
+        a=a,
+    )
+
+    ret = np.sort(to_numpy(ret)).astype(np.complex128)
```
Comment (Aarsh2001) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_linalg/test_matrix_eigenvalues.py:132]: Hey @moamen20, can you please rectify the failing tests. There is no such thing as  `to_numpy`

**Review by Aarsh2001 — CHANGES_REQUESTED** [review]

**Comment by Aarsh2001** [comment]
Hey @a0m0rajab, closing this PR due to inactivity, feel free to reopen once you're done with your changes, thanks

**Pull request closed without merging**